.changeset/cyan-deers-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-node': minor
```

Comment:
Should this be a patch?

Reply:
Yep, makes sense to me, thanks.

https://github.com/backstage/backstage/pull/15613/commits/3c64e308780e0516586e4309d6a217f72dd7cb58